test/functional/wiki_controller_test.rb

```diff
@@ -319,4 +319,10 @@ def teardown
     end
   end
 
+  test "should get wiki with differnt title and path" do
+    wiki = node(:wiki_page)
+    slug = wiki.path.gsub('/wiki/', '')
+    get :show, id: slug
+    assert_response :success
+  end
 end
```

Comment:
OK - I also added this one for root-level wiki pages: https://github.com/publiclab/plots2/commit/1fb8b95c474750706bc265b3f40442656496b664#diff-afcc4c4b1665a46e6c88e913be20b76dR49